packages/travix-ui-kit/components/badge/badge.js

```diff
@@ -103,6 +105,10 @@ Badge.propTypes = {
    * The Badge's position
    */
   position: PropTypes.oneOf(['top', 'right', 'bottom', 'left']),
+  /**
+   * The Badge's size
+   */
+  size: PropTypes.oneOf(['small']),
```

Comment:
Should default be here since it is one of the possible options? (I know that it is the default option, but should to keep its consistency)